e2e/utils/waits-utils.ts

```diff
@@ -263,7 +263,7 @@ export async function waitForText(page: Page,
  */
 export async function waitWhileLoading(page: Page, timeout: number = 90000): Promise<void> {
   const notBlankPageSelector = '[data-test-id="sign-in-container"], title:not(empty), div.spinner, svg[viewBox]';
-  const spinElementsSelector = '[style*="running spin"], .spinner:empty, svg[style*="running rotation"]:not([aria-hidden="true"])';
+  const spinElementsSelector = '[style*="running spin"], .spinner:empty, [style*="running rotation"]:not([aria-hidden="true"])';
```

Comment:
In notebook runtime panel, there is a `img` (spinner) element. remove `svg` so it works for `img` spinner too.